app/src/vanilla/res/menu/fragment_appview.xml

```diff
@@ -36,6 +36,8 @@
 
   <item
       android:id="@+id/menu_remote_install"
+      android:icon="@drawable/ic_download_tv"
+      android:orderInCategory="3"
       android:title="@string/remote_install_menu_title"
-      app:showAsAction="never"/>
+      app:showAsAction="always"/>
```

Comment:
showAsAction="always" overrides the showAsAction="ifRoom" that the share menu icon has. This can make the share icon go to the overflow before the Install on TV icon, making the item order change. This should only happen now tho with the Schedule download + uninstall option, but shouldn't we make sure that it doesn't?

Reply:
I left it this way on purpose so the icon would show. My idea was either forcing "always" on everything, since there are only 3 icons it should be fine, or forcing "ifRoom", this would make everything more scalable and should always show anyway.